src/requests/articles.ts

```diff
@@ -0,0 +1,19 @@
+import { gql } from '@apollo/client';
+
+import client from '../utils/apollo-client';
+import { Article } from '../interfaces/article';
+
+export const getArticles = async (): Promise<Article[]> => {
+    const { data } = await client.query({
+        query: gql`
+            query Articles {
+                articles @rest(type: "Article", path: "/articles") {
```

Comment:
what is type here ? Response type ?

Reply:
Ah I will add that, thank you. Looks like it `ApolloQueryResult` https://www.apollographql.com/docs/react/api/core/ApolloClient/#ApolloClient.query